elements/bulbs-dfp/bulbs-dfp.js

```diff
@@ -48,11 +48,14 @@ export default class BulbsDfp extends BulbsHTMLElement {
   }
 
   handleEnterViewport () {
-    util.getAnalyticsManager().sendEvent({
-      eventCategory: 'bulbs-dfp-element Metrics',
-      eventAction: 'enterviewport',
-      eventLabel: this.dataset.adUnit,
-    });
+    if(!this.trackedEnterViewport) {
```

Comment:
@collin, looks good.  Do we need to reset the value if your :30 seconds logic kicks in to refresh the slot?